### ISSUE TITLE
Able to Override Views, Routes of all the modules

### ISSUE THREAD

**Issue by teddywest32** [AsgardCms/Platform#234]
### Summary of issue

For Now we can only override the views of dashboard and user modules. 
It would be great if we could override all the modules including the Themes for `views` and `routes`.
### System Details
- Operating System `All supported`
- PHP Version `5.6 and greater`
- Laravel Version `5.2 and greater`
- Asgardcms/Platform `version 2.0`
### Easiest Fix

We can add 

```
        $this->publishes([
            __DIR__ . '/../Resources/views' => base_path('resources/views/asgard/user'),
        ]);
        $this->loadViewsFrom(base_path('resources/views/asgard/user'), 'user');
        $this->loadViewsFrom(__DIR__ . '/../Resources/views', 'user');
```

this code in all the modules but it is not a better option. I have tried to add dynamically as suggested by @armababy but no success till now. 
Does anyone have any success on overriding `views` and `routes`?

**Comment by armababy** [comment]
I'll push soon the view override thingy of all AsgardCMS core modules.
About routes i have no idea never looked at that.

**Comment by nWidart** [comment]
Views will be handled by @armababy 's solution. Routes however is more tricky, but you can do it by leveraging how laravel loads its routes: first in first out. You can create the same route but it needs to load before.

**Comment by armababy** [comment]
View override has been merged into 2.0 branch as of b9d45de commit, please try it out and see if it does what you expect.
